packages/agents-manager/src/components/unified-ai-agent/index.tsx

```diff
@@ -74,11 +67,15 @@ function resolveContextEntries( entries: ContextEntry[] ): ContextEntry[] {
 	} );
 }
 
+const queryClient = new QueryClient();
+
 export default function UnifiedAIAgent( props: UnifiedAIAgentProps ) {
 	return (
-		<PersistentRouter>
-			<AgentSetup { ...props } />
-		</PersistentRouter>
+		<QueryClientProvider client={ queryClient }>
```

Comment:
Ensure that the package uses its own React Query Provider.

Reply:
This can be removed. Both Calypso and the Help Center app have providers.